### PR TITLE
Migrate webdev tests to null safety

### PR DESCRIPTION
**Pull request by natebosch**
Towards #1640

### CODE REVIEW

File: webdev/test/e2e_test.dart [hunk 1]
```diff
@@ -309,44 +307,44 @@ void main() {
...
               }));
               Logger.root.fine('vm service uri: $wsUri');
               expect(wsUri, isNotNull);
 
-              vmService = await vmServiceConnectUri(wsUri);
+              vmService = await vmServiceConnectUri(wsUri!);
               var vm = await vmService.getVM();
-              var isolate = vm.isolates.first;
-              var scripts = await vmService.getScripts(isolate.id);
+              var isolate = vm.isolates!.first;
```
Comment (elliette) [inline webdev/test/e2e_test.dart:327]: Looks like we aren't using `isolate` except to get `id` - can we switch the variable to `isolateId = vm.isolates!.first.id!` so we don't have to have all the null-asserts on `isolate.id!` below?

File: webdev/test/e2e_test.dart [hunk 2]
```diff
@@ -372,25 +370,25 @@ void main() {
...
+              vmService = await vmServiceConnectUri(wsUri!);
               var vm = await vmService.getVM();
-              var isolate = await vmService.getIsolate(vm.isolates.first.id);
+              var isolate = await vmService.getIsolate(vm.isolates!.first.id!);
               var library = isolate.rootLib;
 
               await vmService.streamListen('Debug');
 
-              var result = await vmService.evaluate(isolate.id, library.id,
+              var result = await vmService.evaluate(isolate.id!, library!.id!,
```
Comment (elliette) [inline webdev/test/e2e_test.dart:391]: Ditto here - I think `isolateId` and `libraryId` can be pulled out into variables

File: webdev/test/e2e_test.dart [hunk 3]
```diff
@@ -484,27 +482,27 @@ void main() {
...
+              var isolate = await vmService.getIsolate(vm.isolates!.first.id!);
               var library = isolate.rootLib;
 
               await vmService.streamListen('Debug');
 
               expect(
-                  () => vmService.evaluate(
-                      isolate.id, library.id, 'main.toString()'),
+                  () => vmService!
+                      .evaluate(isolate.id!, library!.id!, 'main.toString()'),
```
Comment (elliette) [inline webdev/test/e2e_test.dart:505]: Ditto here, I don't think we use `isolate` and `library` except to get the `id`s

**Review by elliette — APPROVED** [review]
LGTM with a couple small requests. Thank you so much for migrating these tests!

File: webdev/test/e2e_test.dart (hunk 1 above)
Comment (natebosch) [reply webdev/test/e2e_test.dart]: Done

**Pull request merged**